sw/device/lib/crypto/drivers/entropy.c

```diff
@@ -450,9 +450,9 @@ static status_t csrng_send_app_cmd(uint32_t base_address,
         reg = abs_mmio_read32(kBaseCsrng + CSRNG_INTR_STATE_REG_OFFSET);
       } while (!bitfield_bit32_read(reg, CSRNG_INTR_STATE_CS_CMD_REQ_DONE_BIT));
 
-      // Check the "status" bit, which will be 1 only if there was an error.
+      // Check the "status" bit, which will be 0 unless there was an error.
```

Comment:
I'd also combine this commit with the first one, because otherwise SW tests will fail